learn/sherif/main_test/src/main/java/frc/robot/io/Slider.java

```diff
@@ -7,36 +7,45 @@
 import java.util.Map;
 
 public class Slider {
-    private final SimpleWidget m_value;
+    private final SimpleWidget value;
+    private double lastDouble;
+
     public Slider(String name, double defaultValue) {
         this(name, defaultValue, 0, 1.0);
     }
 
     public Slider(String name, double defaultValue, double min, double max) {
-        m_value = BoardManager.getManager().getTab()
+        value = BoardManager.getManager().getTab()
                 .add(name, defaultValue)
                 .withWidget(BuiltInWidgets.kNumberSlider)
                 .withProperties(Map.of("min", min, "max", max));
+        this.lastDouble = defaultValue;
     }
 
     public Slider(String name, double defaultValue, int x, int y, int w, int h) {
         this(name, defaultValue, 0.0, 1.0);
-        m_value.withPosition(x, y)
+        value.withPosition(x, y)
                 .withSize(w, h);
     }
 
     public Slider(String name, double defaultValue, double min, double max, int x, int y, int w, int h) {
         this(name, defaultValue, min, max);
-        m_value.withPosition(x, y)
+        value.withPosition(x, y)
                 .withSize(w, h);
     }
 
     public NetworkTableEntry getEntry() {
-        return m_value.getEntry();
+        return value.getEntry();
     }
 
     public double getDouble(double def) {
-        return this.getEntry()
-                .getDouble(def);
+         var res = this.getEntry()
```

Comment:
The res variable is unnecessary. Instead, just set lastDouble to `this.getEntry().getDouble(def)` and then return lastDouble